packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -91,7 +91,10 @@ class ContainerPlugin {
             if (typeof cacheGroup.chunks === 'function') {
               const prevChunks = cacheGroup.chunks;
               cacheGroup.chunks = (chunk) => {
-                if (chunk.name && chunk.name === name) {
+                if (
+                  chunk.name &&
+                  (chunk.name === name || chunk.name === name + '_partial')
```

Comment:
ensure when runtimeChunk: single is set (and we make a second ContainerPlugin) that the partial container is also excluded from chunk split